src/test/java/com/iota/iri/utils/ConverterTest.java

```diff
@@ -1,6 +1,18 @@
 package com.iota.iri.utils;
 
-public class ConverterTest {
+import org.junit.Assert;
+import org.junit.Test;
 
+public class ConverterTest {
+    @Test
+    public void testFromTextToTrites() {
+        String trytes = "LBCBKBSBCBKBKBRBCCAC9";
+        try {
+            String res = Converter.trytesToAscii(trytes);
+            Assert.assertEquals(res, "B9AI9AAHTR\0");
```

Comment:
Maybe give a better text, to make the test easier to read. 
"EXPECTEDRESULT" will be a nice example :-)